data/wp/wp-content/mu-plugins/EPFL_installs_locked.php

```diff
@@ -7,16 +7,14 @@
  * Author: wwp-admin@epfl.ch
  * */
 
-/* Disable Plugin and Theme Update and Installation
+/* Disable Plugin and Theme Update and Installation.
+ *
+ * Note : this will also set DISALLOW_FILE_EDIT to true.
+ *
  * https://codex.wordpress.org/Editing_wp-config.php
  */
 define( 'DISALLOW_FILE_MODS', true );
 
-/* Disable the Plugin and Theme Editor
- * https://codex.wordpress.org/Editing_wp-config.php
- */
-define( 'DISALLOW_FILE_EDIT', true );
```

Comment:
plus besoin ?

peut-être une note dans la description pour documenter la suppression

(pareil pour ```-add_filter( 'allow_dev_auto_core_updates', '__return_false' );          // Disable development updates```)

Reply:
C'est documenté juste au-dessus:

**Note : this will also set DISALLOW_FILE_EDIT to true.**

Et dans le commit j'ai mis:

As per WordPress documentation : " (i.e. you don't need to set DISALLOW_FILE_MODS and DISALLOW_FILE_EDIT, as on its own DISALLOW_FILE_MODS will have the same effect)."

Pour le add-filter j'ai mis dans le commit:

It's useless (and confusing) to specify allow_dev_auto_core_updates since it's only for development versions

Reply:
ok, parfait pour le commentaire. 

en revanche, les infos dans les commits sont perdues à jamais avec le commit suivant (peut-être même plus tôt 😬 ). 

-> C'est pour ça que nous avons une description dans la PR, avec un template qui est censé t'inciter à remplir ces infos. Tu l'as supprimé ou ça a buggué d'ailleurs ?

Comme les PRs ont aussi une durée de vie limitée, l'idée est de copier-coller cette description dans le CHANGELOG (avec un lien vers la PRs). Je te propose donc de rajouter ça avant de merger 😇

Reply:
> en revanche, les infos dans les commits sont perdues à jamais avec le commit suivant
Euh... non:
https://github.com/erlang/otp/wiki/writing-good-commit-messages

Good commit messages serve at least three important purposes:
* To speed up the reviewing process.
* To help us write a good release note.
* To help the future maintainers of Erlang/OTP (it could be you!), say five years into the future, to find out why a particular change was made to the code or why a specific feature was added

Reply:
il ne reste plus qu'à mettre en oeuvre cette belle théorie dans notre fonctionnement 😵 

😏